src/main/scala/com/netflix/edda/mongo/MongoElector.scala

```diff
@@ -85,7 +85,7 @@ class MongoElector extends Elector {
             "ltime" -> null,
             "data" -> Map("instance" -> instance, "id" -> "leader", "type" -> "leader"))))
       // if we got an error then uniqueness failed (someone else beat us to it)
-      isLeader = if (wr.getN() == 0) true else false
+      isLeader = if (wr.getN() == 1) true else false
```

Comment:
I thought `getN` would only return a count of updated records. Since this is an insert wouldn't it return 0?

https://docs.mongodb.com/manual/reference/command/getLastError/#getLastError.n